### ISSUE TITLE
[Task]: implement liking and commenting on proposals

### ISSUE THREAD

**Issue by elliotBraem** [elliotBraem/devhub-rebuild-with-requests#5]
### Overview

after #2 is integrated, this will be possible.

It's not necessary for the hackathon but would be nice to have for parity.

Reference https://github.com/NEARBuilders/decentralized-social

### Acceptance Criteria

- logged in user can comment
- logged out user cannot
-
